docs/users/contributing-via-pr.md

```diff
@@ -0,0 +1,40 @@
+# Submitting your Operator via Pull Requests (PR) in community operators project
+
+## Overview
+To submit an operator one has to do these steps
+
+1. Fork project `https://github.com/redhat-openshift-ecosystem/community-operators-prod`
+1. Make a pull request
+1. Place the operator in the target directory. [More info](./contributing-where-to.md)
+    - operators
+1. Configure `ci.yaml` file. [More info](./operator-ci-yaml.md)
+    - Setup reviewers
+    - Operator versioning strategy
+1. Verify tests and fix problems, if possible
+1. Ask for help in the PR in case of problems
+
+
+## Pull request
+
+When a pull request is created, a number of tests are executed via community hosted pipeline. One can see the results in the comment section of conversation tab.
+
+![PR](img/op_test_pr.png)
```

Comment:
The img directory is one level up, so this (and other examples within the .md files) needs to be referenced as `../img/..`